src/scripts/Game.ts

```diff
@@ -152,8 +154,8 @@ class Game {
         if (Underground.counter >= GameConstants.UNDERGROUND_TICK) {
             Underground.energyTick(Math.max(0, Underground.energyTick() - 1));
             if (Underground.energyTick() == 0) {
-                Underground.gainEnergy();
-                Underground.energyTick(Underground.getEnergyRegenTime());
+                this.underground.gainEnergy();
+                Underground.energyTick(this.underground.getEnergyRegenTime());
             }
```

Comment:
Should we move this to be in the `Underground` code?
Similar to how the other features handle the ticks with a `tick()` function.